lib/pos_block_producer.go

```diff
@@ -169,6 +182,7 @@ func (pbp *PosBlockProducer) getBlockTransactions(
 		if err != nil {
 			return nil, nil, 0, errors.Wrapf(err, "Error getting transaction size: ")
 		}
+
 		// Skip over transactions that are too big.
 		if currentBlockSize+uint64(len(txnBytes)) > maxBlockSizeBytes {
```

Comment:
Make sure that the mempool cannot accept a txn above this size or it will sit in the mempool forever being skipped by the block producer.

Reply:
Addressed in PR https://github.com/deso-protocol/core/pull/1016